internal/charmstore/debug.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package charmstore
+
+import (
+	"net/http"
+
+	"gopkg.in/juju/charmstore.v4/internal/router"
+	appver "gopkg.in/juju/charmstore.v4/version"
+)
+
+// GET /debug/info .
+func serveDebugInfo(http.Header, *http.Request) (interface{}, error) {
+	return appver.VersionInfo, nil
+}
+
+func newDebug() http.Handler {
```

Comment:
Could we call it newServiceDebug() or something, making sure that it makes a difference from /vX/debug/status?